openedx/features/idea/apps.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
```

Comment:
I see this import in every file, we aren't using it anywhere, is it necessary to keep?

Reply:
https://www.quora.com/What-is-unicode_literals-in-Python